conversions/ipconversion.py

```diff
@@ -0,0 +1,62 @@
+# https://www.geeksforgeeks.org/convert-ip-address-to-integer-and-vice-versa/
+
+
+def ip_to_decimal(ip_address: str) -> int:
+    """
+    Convert an IPv4 address to its decimal representation.
+
+    Args:
+        ip_address (str): A string representing an IPv4 address (e.g., "192.168.0.1").
+
+    Returns:
+        int: The decimal representation of the IP address.
+
+    >>> ip_to_decimal("192.168.0.1")
+    3232235521
+    >>> ip_to_decimal("10.0.0.255")
+    167772415
+    """
+
+    ip_parts = ip_address.split(".")
```

Comment:
See: https://github.com/TheAlgorithms/Python/blob/master/maths/is_ip_v4_address_valid.py
```suggestion
    octets = ip_address.split(".")
```

Reply:
Using more Appropriate variable names indeed make the code easy to understand Thanks!.